install/collecting_ncn_mac_addresses.md

```diff
@@ -70,41 +73,48 @@ For help with either of those, see [LiveCD Setup](bootstrap_livecd_remote_iso.md
     ncn-w003-mgmt
     ```
 
-3. Set the nodes to PXE boot and (re)start them.
+1. Set the nodes to PXE boot and start (or restart) them.
 
     ```bash
```

Comment:
```suggestion
    > `read -s` is used in order to prevent the credentials from being displayed on the screen or recorded in the shell history.

    ```bash
```